node_modules/oae-authentication/tests/test-auth-local.js

```diff
@@ -1173,55 +1205,75 @@ describe('Authentication', function() {
                         assert.ok(err);
                         assert.equal(err.code, 400);
                         assert.ok(!user);
 
                         // Ensure you cannot create a global admin user without a displayName
-                        RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, null, null, function(err, user) {
+                        RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, null, email, {}, function(err, user) {
                             assert.ok(err);
                             assert.equal(err.code, 400);
                             assert.ok(!user);
 
-                            // Ensure the global admin still cannot be authenticated
-                            var testGlobalUserRestContext = TestsUtil.createGlobalRestContext();
-
-                            // Ensure that the credentials do not authenticate a value global administrator
-                            RestAPI.Authentication.login(testGlobalUserRestContext, username, username, function(err) {
+                            // Ensure you cannot create a global admin user without a valid email address
+                            RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, null, null, {}, function(err, user) {
                                 assert.ok(err);
-                                assert.equal(err.code, 401);
+                                assert.equal(err.code, 400);
+                                assert.ok(!user);
+                                RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, null, 'not an email', {}, function(err, user) {
+                                    assert.ok(err);
+                                    assert.equal(err.code, 400);
+                                    assert.ok(!user);
 
-                                // Verify the context was not authenticated
-                                RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
-                                    assert.ok(!err);
-                                    assert.ok(me.anon);
+                                    // Ensure the global admin still cannot be authenticated
+                                    var testGlobalUserRestContext = TestsUtil.createGlobalRestContext();
 
-                                    // Sanity check the user can be created through the REST endpoints
-                                    RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, user) {
-                                        assert.ok(!err);
-                                        assert.ok(user);
+                                    // Ensure that the credentials do not authenticate a value global administrator
```

Comment:
Should this be `Ensure that the credentials do not authenticate a valid global administrator` ?